### PR TITLE
Release: HackerSM64 2.2.0

### PR DESCRIPTION
**Pull request by gheskett**
The time has come.

### CODE REVIEW

**Review by aglab2 — APPROVED** [review]

File: sm64.ld [hunk 1]
```diff
@@ -191,6 +194,9 @@ SECTIONS
       BUILD_DIR/src/audio*.o(.*data*);
 #ifdef S2DEX_TEXT_ENGINE
       BUILD_DIR/src/s2d_engine*.o(.*data*);
+#endif
+#ifdef LIBPL
+      BUILD_DIR/lib/libpl*.o(.*data*);
```
Comment (mountainflaw) [inline sm64.ld:199]: `.*data*`?

File: sm64.ld [hunk 2]
```diff
@@ -224,6 +233,9 @@ SECTIONS
       BUILD_DIR/src/audio*.o(.*bss*);
 #ifdef S2DEX_TEXT_ENGINE
       BUILD_DIR/src/s2d_engine*.o(.*bss*);
+#endif
+#ifdef LIBPL
+      BUILD_DIR/lib/libpl*.o(.*bss*);
```
Comment (mountainflaw) [inline sm64.ld:238]: ditto

**Review by mountainflaw — CHANGES_REQUESTED** [review]

**Review by mountainflaw — APPROVED** [review]
nvm

**Review by Reonu — APPROVED** [review]
MERGE IT

**Review by arthurtilly — APPROVED** [review]
I LOVE PENIS!!!!! AND COCK!!!!!!!!

**Comment by Reonu** [comment]
^ he also loves anime

**Comment by gheskett** [comment]
probably hates games though

**Pull request merged**